src/main/java/eu/stamp_project/testrunner/EntryPoint.java

```diff
@@ -462,6 +462,10 @@ private static String RemoveWinFileSeparator(String string) {
 
     private static String initAbsolutePathToRunnerClasses() {
         URL resource = ClassLoader.getSystemClassLoader().getResource("runner-classes/");
+        //this is the way to make it work from a jenkins plugin
+		if (resource == null) {
+			EntryPoint.class.getClassLoader().getResource("runner-classes/");
```

Comment:
The affectation of `URL resource` is missing, isn't? It should be 
```java
resource = EntryPoint.class.getClassLoader().getResource("runner-classes/");
```
otherwise, the `resource` remains `null`.
also, your indentation seems to be wrong.

Reply:
of course you are right. I copied pasted directly to github editor to speed up things but I missed a piece....
I'll correct it and submit another pull request